packages/website/src/building-blocs/Demo.tsx

```diff
@@ -70,7 +70,7 @@ const Demo = ({children, snippet, center = false, grow = false, title}: DemoProp
                     {title}
                 </Title>
             ) : null}
-            <Group grow className={classes.sandbox} align="stretch" spacing={0}>
```

Comment:
Horizontal layout aren't stretching anymore
 
<img width="1538" alt="image" src="https://user-images.githubusercontent.com/260007/206572070-3917413b-5eb0-4dd9-b908-2596bfc5b0ef.png">

Reply:
Ended up fixing this by replacing <Flex> with <SimpleGrid> and changing the amount of rows depending on layout

Reply:
<img width="1530" alt="image" src="https://user-images.githubusercontent.com/73316533/206745286-7c6cabf0-0f15-49d9-8cd3-cf9946561c48.png">